src/test/java/com/mercadopago/client/order/OrderClientTest.java

```diff
@@ -68,4 +68,51 @@ private static OrderCreateRequest getMinimumOrderCreateRequest() {
                         .build())
                 .build();
     }
+
+    @Test
+    void getSuccess() throws MPException, MPApiException, IOException {
+        HttpResponse response = MockHelper.generateHttpResponseFromFile(CREATE_ORDER_RESPONSE_FILE, HttpStatus.OK);
+        Mockito.doReturn(response).when(HTTP_CLIENT).execute(any(HttpRequestBase.class), any(HttpContext.class));
+
+        String orderId = "123";
+        Order order = client.get(orderId);
+
+        Assertions.assertNotNull(order);
+        Assertions.assertEquals(orderId, order.getId());
+
+    }
```

Comment:
Remover espaço a mais no final
```suggestion
        Assertions.assertEquals(orderId, order.getId());
    }
```